libs/vm-modules/src/math/tensor.cpp

```diff
@@ -28,6 +28,12 @@
 
 using namespace fetch::vm;
 
+namespace fetch {
+namespace vm {
+static ChargeAmount CHARGE_UNIT = 1u;
```

Comment:
Presumably this ChargeAmount should be set somewhere centrally in the vm for re-use in all modules?

Other than this point, this all looks great 👍